src/nfa.js

```diff
@@ -63,6 +63,38 @@ class NFA {
     throw new Error('TODO')
   }
 
+  /**
+   * Checks if some string is contained in the language of this automaton.
+   *
+   * @param  {String} wordInput String to check.
+   *
+   * @return {[type]}           True if it belongs, false otherwise.
+   */
+  match (wordInput) {
+    const word = wordInput.split('').reverse()
+    let states = this.getClosure([ this.start ], '&')
+
+    while (word.length > 0) {
+      const char = word.pop()
+      const nextStates = new Set()
+
+      const x = this.getClosure(states, '&')
```

Comment:
Can we change the name of x to something like statesEpisilonClosure to improve clearity?

Reply:
I saved the values to some variable to print them, for debugging. Forgot to change back to what they were.